spec/features/login/login_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'Login Page', type: :feature do
```

Comment:
- [ ] You should test whether the application redirects to the root page when the correct username and password are entered. 

Example
```ruby
expect(current_path).to eq(root_path)
```

Reply:
> Done 

- [ ]